src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,87 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int VALUE_INDEX = 1;
+    private static final int WORD_INDEX = 0;
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT_VALUE = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File toFile = new File(toFileName);
+        File fromFile = new File(fromFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFile));
+                BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
+            String resultList = getResultText(bufferedReader);
+            bufferedWriter.write(resultList);
+        } catch (IOException e) {
+            throw new RuntimeException("wrong file path");
+        }
```

Comment:
I'd recommend segregating this block into two separate methods: the first one is reading and returning result, next one is writing this result into a file. The main reason for this approvement is to make debugging easier, cause you can customize exceptions for reading and writing (modify message to add file path)